crypto/err/err.c

```diff
@@ -302,8 +318,8 @@ DEFINE_RUN_ONCE_STATIC(do_err_strings_init)
     err_string_lock = CRYPTO_THREAD_lock_new();
     if (err_string_lock == NULL)
         return 0;
-    int_error_hash = lh_ERR_STRING_DATA_new(err_string_data_hash,
-                                            err_string_data_cmp);
+    int_error_hash = lh_ERR_STRING_DATA_new(hash_data,
+                                            cmp_data);
```

Comment:
You could merge these two lines into one, maybe?

Reply:
done.